modules/core/src/test/scala/fs2/kafka/KafkaAdminClientSpec.scala

```diff
@@ -160,10 +160,11 @@ final class KafkaAdminClientSpec extends BaseKafkaSpec {
                 Map(cr -> List(new AlterConfigOp(ce, AlterConfigOp.OpType.SET)))
               }.attempt
               _ <- IO(assert(alteredConfigs.isRight))
-              describedConfigs <- adminClient.describeConfigs(List(cr)).attempt
+              describedConfigs <- adminClient.describeConfigs(List(cr))
               _ <- IO(
                 assert(
-                  describedConfigs.toOption.flatMap(_.get(cr)).map(_.contains(ce)).getOrElse(false)
+                  describedConfigs(cr)
+                    .exists(actual => actual.name == ce.name && actual.value == ce.value)
```

Comment:
The fetched config contains extra metadata which means it no longer compares as equal.